pkgs/top-level/perl-packages.nix

```diff
@@ -18419,10 +18419,9 @@ let
   SysVirt = buildPerlModule rec {
     pname = "Sys-Virt";
     version = "6.3.0";
-    src = assert version == pkgs.libvirt.version; fetchgit {
```

Comment:
Note to reviewers: Sys-Virt-6.3.0 builds correctly with `libvirt-6.6.0` (and should also build with `libvirt-6.7.0`) because there were no API changes in those releases.  Because of this reason there are no Sys-Virt-6.6.0 or Sys-Virt-6.7.0 releases.